Examples/ObjectiveC/UserTrackingModesExample.m

```diff
@@ -95,39 +124,29 @@ - (void)viewDidLoad {
     [self.view addSubview:self.mapView];
     
     [self setupLocationButton];
+    self.mapView.userTrackingMode = MGLUserTrackingModeNone;
```

Comment:
This is the default mode, though I can see the value in showing the developer how to set this.